Author: Daniel Baumann <mail@daniel-baumann.ch>
Description: Avoid overwriting build-environment.

diff -Naurp lzlib.orig/configure lzlib/configure
--- lzlib.orig/configure
+++ lzlib/configure
@@ -26,9 +26,9 @@ infodir='$(datarootdir)/info'
 libdir='$(exec_prefix)/lib'
 mandir='$(datarootdir)/man'
 CC=gcc
-CPPFLAGS=
-CFLAGS='-Wall -W -O2'
-LDFLAGS=
+#CPPFLAGS=
+#CFLAGS='-Wall -W -O2'
+#LDFLAGS=
 
 # checking whether we are using GNU C.
 ${CC} --version > /dev/null 2>&1
